core/components/footer/locale.tsx

```diff
@@ -0,0 +1,19 @@
+import { Globe } from 'lucide-react';
+import { getLocale } from 'next-intl/server';
+
+import { locales } from '~/i18n';
+
+import { Link } from '../link';
+
+export const Locale = async () => {
+  const locale = await getLocale();
+
+  return (
+    // eslint-disable-next-line @typescript-eslint/no-unnecessary-condition
+    locales.length > 1 && (
```

Comment:
Want to hide this link if there is only one locale, unfortunately had to prevent this rule from applying because when it is only one locale typescript doesn't see the need for this conditional.